tracardi/process_engine/action/v1/sort_action.py

```diff
@@ -0,0 +1,75 @@
+from pydantic import validator
+from tracardi.service.plugin.domain.config import PluginConfig
+from tracardi.domain.event import Event
+from tracardi.domain.session import Session
+from tracardi.domain.profile import Profile
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.process_engine.tql.equation import MathEquation
+
+
+class SortConfig(PluginConfig):
+    data: str
+    direction: str
+
+    @validator("data")
+    def must_not_be_empty(cls, value):
+        if len(value) == 0:
+            raise ValueError("data is empty")
+        return value
+
+    @validator("direction")
+    def must_not_be_empty(cls, value):
+        if value not in ("asc", "desc"):
+            raise ValueError("direction is not \"asc\" or \"desc\"")
+        return value
+
+
+def validate(config: dict):
+    return SortConfig(**config)
+
+
+class SortAction(ActionRunner):
+    config: SortConfig
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None) -> Result:
+        reverse = True if self.config.direction == "desc" else False
+        array = sorted(payload[self.config.data], reverse=reverse)
+        return Result(port="sorted_array", value=array)
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        debug=False,
+        spec=Spec(
+            module='tracardi.process_engine.action.v1.sort',
```

Comment:
It it better to put __name__ here this way when we refactor (move the location of the plug-in) it will not break.